go-controller/pkg/ovn/gateway_cleanup.go

```diff
@@ -128,15 +128,20 @@ func delPbrAndNatRules(nodeName string) {
 			"stdout: %s, stderr: %s, error: %v", matches, stderr, err)
 		matches = ""
 	}
-	nodeSubnetMatchSubStr := fmt.Sprintf("rtos-%s", nodeName)
+	// matches inport for a policy, must use ending quote to substring match to avoid matching other nodes accidentally
+	// i.e. rtos-ovn-worker would match rtos-ovn-worker2
+	nodeSubnetMatchSubStr := fmt.Sprintf("rtos-%s\"", nodeName)
```

Comment:
I mean, in theory you might have one node called "worker" and another node called "albertos-worker"... so include the starting quote too?

Reply:
@danwinship what you are saying will not be possible right. Since, you will have `rtos-worker` and `rtos-albertos-worker`,

Reply:
@trozet the output looks like this:

```
_uuid               : 3eff8033-7f48-4b1d-98f6-c74994a58142
action              : reroute
external_ids        : {}
match               : "inport == \"rtos-k8s-master-01\" && ip4.dst == 10.0.0.60 /* k8s-master-01 */"
nexthop             : "10.192.2.2"
priority            : 1004
```

So, shouldn't we have

`fmt.Sprintf(`rtos-%s\"`, nodeName)`

Reply:
because of data=bare, there wont be any backslash